PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -47,32 +50,92 @@ enum V0Type : uint8_t {
   Lambda,
   AntiLambda
 };
-const int nBins = 7;
-constexpr double binsPt[nBins + 1] = {
+
+const int nBinsPt = 7;
+constexpr double binsPt[nBinsPt + 1] = {
   1.,
   2.,
   4.,
   6.,
   8.,
   12.,
   24.,
-  50.};
-auto vecBins = std::vector<double>{binsPt, binsPt + nBins + 1};
+  1000.};
+auto vecBinsPt = std::vector<double>{binsPt, binsPt + nBinsPt + 1};
+
+namespace hf_cuts_D_daughter
+{
+const int nBinsPtD = 7;
+static constexpr int nCutVarsD = 6;
+constexpr double binsPtD[nBinsPtD + 1] = {
+  1.,
+  2.,
+  4.,
+  6.,
+  8.,
+  12.,
+  24.,
+  1000.};
+auto vecBinsPtD = std::vector<double>{binsPtD, binsPtD + nBinsPtD + 1};
+// default values for the cuts
+constexpr double cutsD[nBinsPtD][nCutVarsD] = {{1.84, 1.89, 1.77, 1.81, 1.92, 1.96},  /* 1   < pt < 2 */
+                                               {1.84, 1.89, 1.77, 1.81, 1.92, 1.96},  /* 2 < pt < 4 */
+                                               {1.84, 1.89, 1.77, 1.81, 1.92, 1.96},  /* 4   < pt < 6 */
+                                               {1.84, 1.89, 1.77, 1.81, 1.92, 1.96},  /* 6 < pt < 8 */
+                                               {1.84, 1.89, 1.77, 1.81, 1.92, 1.96},  /* 8 < pt < 12 */
+                                               {1.84, 1.89, 1.77, 1.81, 1.92, 1.96},  /* 12   < pt < 24 */
+                                               {1.84, 1.89, 1.77, 1.81, 1.92, 1.96}}; /* 24   < pt < 1000 */
+// row labels
+static const std::vector<std::string> labelsPt{};
+// column labels
+static const std::vector<std::string> labelsCutVarD = {"invMassSignalLow", "invMassSignalHigh", "invMassLeftSBLow", "invMassLeftSBHigh", "invMassRightSBLow", "invMassRightSBHigh"};
+} // namespace hf_cuts_D_daughter
+
+namespace hf_cuts_V0_daughter
+{
+const int nBinsPtV0 = 7;
+static constexpr int nCutVarsV0 = 5;
+constexpr double binsPtV0[nBinsPtV0 + 1] = {
+  0.,
+  1.,
+  2.,
+  4.,
+  8.,
+  12.,
+  24.,
+  1000.};
+auto vecBinsPtV0 = std::vector<double>{binsPtV0, binsPtV0 + nBinsPtV0 + 1};
+// default values for the cuts
+constexpr double cutsV0[nBinsPtV0][nCutVarsV0] = {{0.48, 0.52, 0.99, 1., 0.9},  /* 1   < pt < 2 */
+                                                  {0.48, 0.52, 0.99, 1., 0.9},  /* 2 < pt < 4 */
+                                                  {0.48, 0.52, 0.99, 1., 0.9},  /* 4   < pt < 6 */
+                                                  {0.48, 0.52, 0.99, 1., 0.9},  /* 6 < pt < 8 */
+                                                  {0.48, 0.52, 0.99, 1., 0.9},  /* 8 < pt < 12 */
+                                                  {0.48, 0.52, 0.99, 1., 0.9},  /* 12   < pt < 24 */
+                                                  {0.48, 0.52, 0.99, 1., 0.9}}; /* 24   < pt < 1000 */
+// row labels
+static const std::vector<std::string> labelsPt{};
+// column labels
+static const std::vector<std::string> labelsCutVarV0 = {"invMassLow", "invMassHigh", "cpaMin", "dcaMax", "radiusMin"};
+} // namespace hf_cuts_V0_daughter
```

Comment:
It would be good to move this to a header file, such as [SelectorCuts.h](https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/Core/SelectorCuts.h in PWGHF/Core or to create an equivalent one in the D2H folder)